gradle/versions.gradle

```diff
@@ -17,8 +17,8 @@ ext {
     vision_ar = vision
     vision_safety = vision
 
-    mapbox_core = '0.2.1'
-    mapbox_telemetry = '3.5.8'
+    mapbox_core = '1.3.0'
+    mapbox_telemetry = '4.5.1'
     mapbox_navigation = '0.22.0'
```

Comment:
Lets remove mapbox_navigation and other versions unused here

Reply:
@yunikkk can't remove them because they are used by _Examples_, updated their versions to the latest ones